internal/config/options.go

```diff
@@ -94,6 +94,10 @@ type Options struct {
 
 	Policies []policy.Policy
 
+	// Administrators contains a set of emails with users who have super user
+	// (sudo) access including the ability to impersonate other users' access
+	Administrators []string `envconfig:"ADMINISTRATORS"`
```

Comment:
Options fields need the 'mapstructure' tag to work with viper and should be LC'd going forward.